### PR TITLE
Include Support For RVT Script

### PR DESCRIPTION
**Pull request by Sopitive**
New Language Grammar

### Changes

Added RVT Script Language Grammar

### Checklist
- [X] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: SUPPORTED_LANGUAGES.md [hunk 1]
```diff
@@ -184,6 +184,7 @@ The table below shows the full list of languages (and corresponding classes/alia
 | RPM spec files          | rpm-specfile, rpm, spec, rpm-spec, specfile | [highlightjs-rpm-specfile](https://github.com/highlightjs/highlightjs-rpm-specfile) |
 | Ruby                    | ruby, rb, gemspec, podspec, thor, irb | |
 | Rust                    | rust, rs               |         |
+| RVT Script              | rvt, rvt script        | [highlightjs-rvt-script](https://github.com/Sopitive/highlightjs-rvt-script) |
```
Comment (joshgoebel) [inline SUPPORTED_LANGUAGES.md:187]:
```suggestion
| RVT Script              | rvt, rvt-script        | [highlightjs-rvt-script](https://github.com/Sopitive/highlightjs-rvt-script) |
```

Do you have an alias for `rvt` in your grammar though?  I'm not sure I saw one.

**Comment by joshgoebel** [comment]
Changeling entry as well, please. Thanks!

**Comment by Sopitive** [comment]
Updated the changelog and and added the alias.

File: CHANGES.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ New Grammars:
 - added 3rd party LookML grammar to SUPPORTED_LANGUAGES [Josh Temple][]
 - added 3rd party FunC grammar to SUPPORTED_LANGUAGES [Nikita Sobolev][]
 - Added 3rd party Flix grammar to SUPPORTED_LANGUAGES [The Flix Organisation][]
+- Added 3rd party RVT grammar to SUPPORTED_LANGUAGES [Sopitive]
```
Comment (joshgoebel) [inline CHANGES.md:8]:
```suggestion
- Added 3rd party RVT grammar to SUPPORTED_LANGUAGES [Sopitive][]
```

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**